.github/workflows/close-old-issues.yaml

```diff
@@ -10,14 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       issues: write  # Ensure necessary permissions for issues
-      pull-requests: none
-      contents: none
     steps:
     - name: Close inactive issues
       uses: actions/stale@v9.0.0
       with:
+        # A list of labels to reference when looking through issues,
+        # and only when one (or even more) of these labels are found..
+        # then skip this issue, and never try to stale and/or close it.
         exempt-issue-labels: "Keep Issue Open"
-        days-before-issue-close: 14
-        close-issue-message: "This issue was closed because it has been inactive for 14 days"
-        debug-only: false # Make this field equal true if you want to test your configuration if it works or not
+        # Split it into two weeks, after one week the issue will be marked as stale,
+        # after another week have pasted without any update.. the issue will then be closed.
+        days-before-issue-stale: 7
+        days-before-issue-close: 7
+        # NEVER mark PRs as Stale or Close + this workflow should never have write permissions on PRs, EVER!
```

Comment:
I've written this because making a workflow have the ability to close any inactive PR (after a set period of time) is just.. it doesn't make any sense to me! At least in the context of WinUtil Repo + this workflow is designed for closing old issues (as file name suggests).

if anyone can find a compelling argument as to why we should expand the functionality of this workflow to close old PRs (with a period of time the same or different from issues).. then please go ahead and do so.